src/makeCalculator.js

```diff
@@ -38,6 +38,47 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    'result': 0,
+
+    add(num, that) {
+      that.result += num;
+
+      return that;
+    },
+
+    subtract(num, that) {
+      that.result -= num;
+
+      return that;
+    },
+
+    divide(num, that) {
+      that.result /= num;
```

Comment:
можна було об'єкт calculator тут використати
```suggestion
      calculator.result /= num;
```

Reply:
Спасибо, я так и понял что можно было сделать как-то по-другому.